frontend/src/app/testQueue/AoEForm/AoEModalForm.jsx

```diff
@@ -101,39 +101,54 @@ const AoEModalForm = ({
       <h2 className="font-heading-lg margin-top-205 margin-bottom-0">
         Test questionnaire
       </h2>
```

Comment:
I think this header also needs to be behind the feature flag

Reply:
Okay, moved that in as well